client/components/Shop/Products/Products.jsx

```diff
@@ -16,26 +19,83 @@ const ProductList = ({ currentCategory }) => {
   };
   useEffect(getProducts, []);
 
+  const ExampleStyledComponent = styled.div`
```

Comment:
Just for code cleanliness- would be nice if everything is aligned similarly.

Reply:
Agreed - I initially declared this styled div as ExampleStyledComponent while playing with Styled Components, but I will update.